### PR TITLE
New servereless pattern eventbridge-pipes-amq-to-redshift

### PR DESCRIPTION
**Pull request by vinodh-aws**
*Issue #2052 

*Description of changes:*
Description of changes:
This pattern demonstrates how to send messages from Amazon MQ for ActiveMQ queue to Amazon Redshift cluster using Amazon EventBridge Pipes and utilizing Amazon Redshift data API queries as target.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

**Review by ellisms — CHANGES_REQUESTED** [review]
Minor suggested changes.

File: eventbridge-pipes-sqs-to-redshift/example-pattern.json [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+    "text": [
+      "See the README in the Github repo for detailed testing instructions."
+    ]
+  },
+  "cleanup": {
+    "text": [
+      "Delete the stack: <code>cdk destroy</code>."
+    ]
+  },
+  "authors": [
```
[the remark this answers is not in the document]
Comment (vinodh-aws) [reply eventbridge-pipes-sqs-to-redshift/example-pattern.json:60]: Hi ellisms@  

Yes, I'm helping Rajat as a part of Messaging Focus Area

**Comment by vinodh-aws** [comment]
Hi ellisms@ Checking you have any further questions in this request ?

**Comment by ellisms** [comment]
Hi @vinodh-aws  - no additional questions. I'll start working on the pre-publishing tasks. Thanks!

**Comment by boyney123** [comment]
Looks like the pull request has merge conflicts, can you resolve these? Thanks!

**Comment by ellisms** [comment]
@boyney123 Can you try now? If you still get a conflict on that file I think we can drop it completely, the serverlessland json to use file is `eventbridge-pipes-amq-to-redshift-cdk-python/ebpipes-amq-redshift-cdk-python.json`

**Pull request merged**